src/styles/index.scss

```diff
@@ -1,3 +1,104 @@
+* {
+  box-sizing: border-box;
+}
+
+@import 'vars';
+@import 'stars';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+}
+
 body {
   margin: 0;
+  text-decoration: none;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
```

Comment:
```suggestion
```

Reply:
This part was suggested by mentors in group chat. 
If I change it, my test will be failed

Reply:
Still not fixed. Please feel free to ask for help in fe_chat to remove this hardcoded height.